pyocd/subcommands/rtt_cmd.py

```diff
@@ -24,6 +24,70 @@
 from pyocd.utility.cmdline import convert_session_options, int_base_0
 from ctypes import Structure, c_char, c_int32, c_uint32, sizeof
 
+import os
+# Windows
+if os.name == 'nt':
+    import msvcrt
+# Posix (Linux, OS X)
+else:
+    import sys
+    import termios
+    import atexit
+    from select import select
+
+class KBHit:
+    # taken from https://stackoverflow.com/questions/2408560/non-blocking-console-input
+
+    def __init__(self):
+        '''Creates a KBHit object that you can call to do various keyboard things.
+        '''
+
+        if os.name == 'nt':
+            pass
+        else:
+            # Save the terminal settings
+            self.fd = sys.stdin.fileno()
+            self.new_term = termios.tcgetattr(self.fd)
+            self.old_term = termios.tcgetattr(self.fd)
+
+            # New terminal setting unbuffered
+            self.new_term[3] = (self.new_term[3] & ~termios.ICANON & ~termios.ECHO)
+            termios.tcsetattr(self.fd, termios.TCSAFLUSH, self.new_term)
+
+            # Support normal-terminal reset at exit
+            atexit.register(self.set_normal_term)
+
+    def set_normal_term(self):
+        ''' Resets to normal terminal.  On Windows this is a no-op.
+        '''
+
+        if os.name == 'nt':
+            pass
+        else:
+            termios.tcsetattr(self.fd, termios.TCSAFLUSH, self.old_term)
+
+    def getch(self):
+        ''' Returns a keyboard character after kbhit() has been called.
+            Should not be called in the same program as getarrow().
+        '''
+
+        s = ''
```

Comment:
This is the unused local variable identified by LGTM.

Reply:
Ooh, good catch on the license! Thanks for following up on that!

Reply:
@simondlevy kindly agreed to re-license the KBHit class under an MIT License (which is Apache 2.0 compatible) for inclusion in pyOCD. I'll go ahead and add this information to the copyright notice, adding his name too.

Reply:
All should be OK now.

Reply:
@simondlevy made available the code on GitHub under the MIT License. I've updated the comments accordingly.

Reply:
Exellent! 😄 

We should probably put KBHit in its own `pyocd.utility.kbhit.py` module, so the whole file is MIT-licensed and has his copyright. (I'll also add this comment on the `KBHit` class in the updated `rtt_cmd.py` file.)

Reply:
Done.

Reply:
Thanks!